instance/models/openedx_appserver.py

```diff
@@ -53,6 +53,21 @@
 ]
 
 
+class ProvisioningError(Exception):
+    """
+    Raised when an provisioning fails.
+    """
+
+    def __init__(self, message, log=None):
```

Comment:
@kewne Argument `log` has not been used anywhere. Can we just skip it and simplify the error class to -

```python
class ProvisioningError(Exception):
    """
    Raised when an provisioning fails.
    """
    pass
```

Reply:
Ah, you're right: my intent was to also use it [here](https://github.com/open-craft/opencraft/blob/419e0aa32072a36eb5c83b6d80154fb625c4fefd/instance/models/openedx_appserver.py#L718) so it would become something like
```python
if exit_code != 0:
    self._status_to_configuration_failed()
    self.manage_instance_services(active=False)
    raise ProvisionError("AppServer deploy failed: Ansible play exited with non-zero exit code", log)
```

What do you think? I've also realized it'd be best to document the intended use of `ProvisionError` in these methods.

Reply:
@kewne, Ah! okay, I didn't see that use case. Yes, I think we can use the new `ProvisionError` there and document it properly.